testing/mocks/generic.go

```diff
@@ -201,32 +200,67 @@ func GenericLedgerPath(index int) ledger.Path {
 	return GenericLedgerPaths(index + 1)[index]
 }
 
-func GenericLedgerValues(number int) []ledger.Value {
+func GenericRegisterEntries(number int) flow.RegisterEntries {
+	registers := make(flow.RegisterEntries, 0, number)
+	for i := 0; i < number; i++ {
+		registers = append(registers,
+			flow.RegisterEntry{
+				Key:   GenericRegister(i),
+				Value: GenericRegisterValue(i),
+			})
+	}
+
+	return registers
+}
+
+func GenericRegisterEntry(index int) flow.RegisterEntry {
+	return GenericRegisterEntries(index + 1)[index]
+}
+
+func GenericRegisters(number int) flow.RegisterIDs {
 	// Ensure consistent deterministic results.
 	random := rand.New(rand.NewSource(3))
 
-	var values []ledger.Value
+	registers := make(flow.RegisterIDs, 0, number)
 	for i := 0; i < number; i++ {
-		value := make(ledger.Value, 32)
-		binary.BigEndian.PutUint64(value[0:], random.Uint64())
-		binary.BigEndian.PutUint64(value[8:], random.Uint64())
-		binary.BigEndian.PutUint64(value[16:], random.Uint64())
-		binary.BigEndian.PutUint64(value[24:], random.Uint64())
+		registers = append(registers, flow.RegisterID{
```

Comment:
this should call GenericRegister, and `flow.RegisterID{...}` should be in GenericRegister's body.

Reply:
Oh, you are breaking my hear^W^W this tests' quadratic, cubic, quatric, etc, complexity Patrick!